packages/ogm/package.json

```diff
@@ -29,8 +29,7 @@
         "@graphql-tools/merge": "^6.2.13",
         "@neo4j/graphql": "^1.0.2",
         "camelcase": "^6.2.0",
-        "pluralize": "^8.0.0",
-        "upper-case-first": "^2.0.2"
+        "pluralize": "^8.0.0"
```

Comment:
We should do the same as upper first with the OGM: WDYT? 

```
import { Neo4jGraphQL, upperFirst } from "@neo4j/graphql";
```

Reply:
`graphql-compose` isn't a dependency of the OGM (despite the fact it will always be there due to dependency on `@neo4j/graphql`) and ESLint rightfully complains, so the solution here for the OGM is to get around that fact.